### PR TITLE
Vam 43 (setSize), 86 (setSort), 409 (saveSize)

### PR DESCRIPTION
**Pull request by sidjtd**
Complete

### CODE REVIEW

File: emulation/models/orwch-savesiz.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        id: 'ORWCH_SAVESIZ',
+        name: 'ORWCH SAVESIZ',
+        function: 'SAVESIZ^ORWCH',
+        parameters: ['ORWCH BOUNDS'],
+        files: [],
+        domain: 'CPRS',
+        labels: ['META', 'ORDER', 'CHANGE'],
+        services: ['parameterService.addChange'],
+        rpcType: 'CHANGE',
+        complexity: 'MEDIUM',
```
Comment (mfuroyama) [inline emulation/models/orwch-savesiz.js]: Not a huge deal, but I'd say let's change this to 'LOW'; this one was a gimme... 😄

File: emulation/models/orworb-setsort.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    .then((res) => {
+                        const response = res || 0;
+                        callback(null, response);
+                    }).catch(err => callback(err));
+            }
+
+            // If first argument is invalid, return validation logic error
+            if (!_.isString(sort) || _.isEqual(sort, '')) return callback(null, errorString);
+            const sortValue = sort[0].toUpperCase();
+            if (sortValue.length > 1 || !sortValue.match(/P|M|U|I|L|D|F/)) return callback(null, errorString);
```
Comment (mfuroyama) [inline emulation/models/orworb-setsort.js]: Consider replacing the regex here:
```
/P|M|U|I|L|D|F/
```
with the regex in its more common form:
```
/[PMUILDF]/
```

File: emulation/models/orworb-setsort.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+            // If there is a second argument, use sortDir instead as the paramValue
+            if (_.isString(dir)) {
+                const sortDir = dir.toUpperCase();
+                if (sortDir.length > 1 || !sortDir.match(/F|R/)) return callback(null, errorString);
```
Comment (mfuroyama) [inline emulation/models/orworb-setsort.js]: Consider replacing the regex here:
```
/F|R/
```
with the regex in its more common form:
```
/[FR]/
```

File: emulation/models/tests/orwch-savfont-spec.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const rpcEmulatorModel = require('../orwch-savfont').rpcEmulatorModel;
+const unitTestUtils = require('./unitTestUtils');
+
+describe('ORWCH SAV FONT asynchronous specs', () => {
```
Comment (mfuroyama) [inline emulation/models/tests/orwch-savfont-spec.js]: For pedantic reasons, could we change this to `ORWCH SAVFONT`? Thanks!

File: emulation/models/tests/orworb-setsort-spec.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const rpcEmulatorModel = require('../orworb-setsort').rpcEmulatorModel;
+const unitTestUtils = require('./unitTestUtils');
+
+describe('ORWORBSETSORT asynchronous specs', () => {
```
Comment (mfuroyama) [inline emulation/models/tests/orworb-setsort-spec.js]: Same here...could we change this to `ORWORB SETSORT` to match the name of the RPC? Thanks!

Comment (sidjtd) [reply emulation/models/tests/orworb-setsort-spec.js]: You got it!

**Pull request merged**